packages/convex-helpers/server/customFunctions.test.ts

```diff
@@ -62,7 +62,11 @@ const authenticatedQueryBuilder = customQuery(
 type AuthQueryCtx = CustomCtx<typeof authenticatedQueryBuilder>;
 
 // Example query that doesn't specify argument validation (no `args` param).
-export const unvalidatedArgsQuery = authenticatedQueryBuilder((ctx) => {
+export const unvalidatedArgsQuery: RegisteredQuery<
```

Comment:
This is to solve the following error:

```
The inferred type of 'addCU' cannot be named without a reference to '../../../node_modules/convex/dist/esm-types/server/registration.js'. This is likely not portable. A type annotation is necessary. ts(2742)
```

This error could also be solved by removing the `export`, but this is is not a problem since we’re in a test file.